docs/modules/ROOT/pages/runbooks/ManyBillingEntitiesCreated.adoc

```diff
@@ -0,0 +1,38 @@
+= Alert rule: ManyBillingEntitiesCreated
+
+include::partial$runbooks/contribution_note.adoc[]
+
+== icon:glasses[] Overview
+
+This alert fires if many BillingEntities have been created.
+This is usually the case if someone tries to abuse APPUiO Cloud.
+
+== icon:bug[] Steps for debugging
+
+
+=== Check the newly created BillingEntities
+
+.List all billing entities created less than 5 minutes ago
+[source,shell]
+----
+start=$(date -u -d '5 min ago' '+%Y-%m-%dT%H:%m:%SZ')
```

Comment:
Maybe 5 minutes are a bit short? It may take a while to react on an alert.

But I also see runbook instructions more as a template and not literal copy pasting without any thought involved 😅

Reply:
Fair enough, I can adjust the provided command to 30 minutes or so